src/Security/MemberAuthenticator/LoginForm.php

```diff
@@ -26,7 +31,7 @@
  *    allowing extensions to "veto" execution by returning FALSE.
  *    Arguments: $member containing the detected Member record
  */
-class MemberLoginForm extends LoginForm
+class LoginForm extends BaseLoginForm
```

Comment:
This class now differs from LoginForm only by namespace; I suggest leaving this as MemberLoginForm to avoid ambiguity.